cabal.project

```diff
@@ -69,3 +69,29 @@ if(impl(ghc >= 9.6.1))
 
   package servant-server
     ghc-options: -fprint-redundant-promotion-ticks
+
+allow-newer: postgresql-simple:base
+allow-newer: postgresql-simple:template-haskell
+allow-newer: postgresql-simple:containers
+allow-newer: postgresql-libpq:base
+allow-newer: servant-swagger-ui:base
+allow-newer: servant-swagger-ui-core:base
+allow-newer: file-embed-lzma:base
+allow-newer: file-embed-lzma:template-haskell
+allow-newer: file-embed-lzma:filepath
+allow-newer: lzma:base
```

Comment:
This one was already bumped on hackage: https://hackage.haskell.org/package/lzma